lib/Mojo/Message/Request.pm

```diff
@@ -197,6 +197,8 @@ sub _parse_env {
     $base->path->parse($value =~ m!/$! ? $value : "$value/");
 
     # Remove SCRIPT_NAME prefix if necessary
+    # Make sure $path is decoded and unescaped as expected by to_string
```

Comment:
Reads like you're putting things that should be in a nice detailed commit message into comments.